src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -48,4 +48,5 @@ Previously deprecated classes in the `spring-integation-hazelcast` module, such
 [[x7.0-mqtt-changes]]
 === MQTT Changes
 
-The `disconnectForcibly` method in `MqttPahoMessageDrivenChannelAdapter`, `Mqttv5PahoMessageDrivenChannelAdapter`, `Mqttv3ClientManager`, and `Mqttv5ClientManager` now supports `getQuiescentTimeout` in addition to the existing `disconnectCompletionTimeout`.
+The `disconnectForcibly` method in `MqttPahoMessageDrivenChannelAdapter`, `Mqttv5PahoMessageDrivenChannelAdapter`, `Mqttv3ClientManager`, and `Mqttv5ClientManager` now expose the option to configure `quiescentTimeout`, which is available through the https://eclipse.dev/paho/files/javadoc/org/eclipse/paho/client/mqttv3/IMqttAsyncClient.html[IMqttAsyncClient] client.
```

Comment:
Sorry, I still disagree with this sentence.
How about like this:
```
The `AbstractMqttMessageDrivenChannelAdapter` and `ClientManager` implementations now expose a `quiescentTimeout` option which is propagated in their `stop()` method down to the `disconnectForcibly()` API of the MQTT Paho clients.
```
The JavaDoc to MQTT v3 client is not quit right here since we also do the same for MQTT v5 client interaction.

Reply:
Should we add MQTT V3 and V5 links.   OR remove the V3?

Reply:
Remove altogether.
We may mention something like `See their JavaDocs for more information.`, though.